data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
+    "description": "Gun in a cane.  A custom-built .44 Magnum rifle carefully integrated into a walking cane.  It's also reinforced for hand-to-hand combat.  Never underestimate a handicapped person.",
+    "weight": 950,
+    "volume": 4,
+    "price": 400000,
+    "bashing": 15,
+    "to_hit": 2,
+    "material": [ "steel", "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "dark_gray",
+    "skill": "rifle",
+    "ammo": "44",
+    "ranged_damage": 11,
```

Comment:
A quick glance at Wikipedia turned up this (on the cane gun page, go
figure):
> Modern cartridge-type cane guns are usually fitted to fire large
low-pressure (black powder or equivalent) handgun ammunition or shotgun
cartridges between .410 like the 12-gauge, both of which are well-suited
use in a weapon that is effectively just a barrel with integrated chamber,
manual ejection, a detachable a firing mechanism, a rudimentary grip and
(in some cases) even more rudimentary sights.

As I suspected, a .45 round is the right calibre, but this kind of gun only
accepts low-pressure versions of the round. So it needs specialized ammo
and will cause less damage than other .45 guns, not more, though that can
be modelled in the ammo itself.